internal/types.go

```diff
@@ -146,6 +151,14 @@ type MediaIds struct {
 	Ids []string `json:"ids"`
 }
 
+type ExternalIds struct {
+	ImdbId string `json:"imdb_id"`
+}
+
+func getImdbID(externalIds ExternalIds) string {
+	return externalIds.ImdbId
+}
```

Comment:
Not needed if the above suggestion works

Reply:
Done